src/main/java/core/basesyntax/Area.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Area {

```

Comment:
Don't begin class or method implementation with an empty line. Please remove the redundant empty line at the beginning of the interface declaration.